src/Sylius/Bundle/CoreBundle/Import/Writer/ORM/Processor/UserProcessor.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Import\Writer\ORM\Processor;
+
+use Sylius\Component\ImportExport\Converter\DateConverterInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class UserProcessor implements UserProcessorInterface
```

Comment:
Maybe we could use one processor for both import and export. We can have two methods: ``` convert ``` and ``` revert ```. WDYT?